pkgs/racket-doc/scribblings/reference/eval-model.scrbl

```diff
@@ -169,15 +171,16 @@ then
 #,(redex (+ x 1)) #,reduces #,(redex (+ 10 1)) #,reduces 11
 ]
 
-In Racket, the way definitions appear is just as important as the way
-that they are used. Racket evaluation thus keeps track of both
+In Racket, the way definitions are created is just as important as the way
+they are used. Racket evaluation thus keeps track of both
 definitions and the current expression, and it extends the set of
 definitions in response to evaluating forms such as @racket[define].
 
-Each evaluation step, then, takes the current set of definitions and
-program to a new set of definitions and program. Before a
+Each evaluation step, then, transforms the current set of definitions and
+program into a new set of definitions and program. Before a
 @racket[define] can be moved into the set of definitions, its
-right-hand expression must be reduced to a @tech{value}.
+second expression must be reduced to a @tech{value}.
+(The first expression is used without evaluation.)
```

Comment:
This is incorrect, since definitions only have *one* expression, the right-hand side. In `(define x e)`, `x` is not an expression at all but merely an identifier, so the only expression is `e`. I personally think the original wording is alright (since this section is not intended to document the `define` form, by any means), but if you wanted to make it clearer, you could reword it to “its expression—that is, its right-hand side—must be reduced”.